addons/api/addon/generated/models/role.js

```diff
@@ -28,6 +28,12 @@ export default class GeneratedRoleModel extends BaseModel {
   })
   grant_scope_id;
```

Comment:
Will we still use this `grant_scope_id` or will that be replaced with `grant_scope_ids`?

Reply:
`grant_scope_id` but that will be done later when we remove all references to that attribute.